pom.xml

```diff
@@ -550,6 +550,20 @@
     </dependencies>
   </dependencyManagement>
 
+  <repositories>
```

Comment:
Why do we need this now?

Reply:
leftover from using Iceberg snapshot version. removed